### PR TITLE
PTX-1981 Add upgrade PX Operator version methods and test

### PR DESCRIPTION
**Pull request by nikolaypopov**
Add upgrade Operator version methods
Add test to perform upgrade hops for Operator

Signed-off-by: nikolaypopov <nikolay.popov86@gmail.com>

### CODE REVIEW

File: test/integration_test/utils/px_operator.go [hunk 1]
```diff
@@ -60,3 +63,60 @@ func getPXOperatorImageTag() (string, error) {
...
+}
+
+// GetPxOperatorDeployment return PX Operator deployment
+func GetPxOperatorDeployment() (*appsv1.Deployment, error) {
+	pxOperator, err := appops.Instance().GetDeployment(PortworxOperatorDeploymentName, PxNamespace)
+	if err != nil {
+		return nil, err
+	}
+
+	return pxOperator, nil
```
Comment (piyush-nimbalkar) [inline test/integration_test/utils/px_operator.go:93]: Why not just write it this way - 
```
    return appops.Instance().GetDeployment(PortworxOperatorDeploymentName, PxNamespace)
```

File: test/integration_test/utils/px_operator.go [hunk 2]
```diff
@@ -60,3 +63,60 @@ func getPXOperatorImageTag() (string, error) {
 
 	return strings.Split(image, ":")[1], nil
 }
+
+// GetPxOperatorImage return PX Operator image
+func GetPxOperatorImage() (string, error) {
+	deployment, err := appops.Instance().GetDeployment(PortworxOperatorDeploymentName, PxNamespace)
```
Comment (piyush-nimbalkar) [inline test/integration_test/utils/px_operator.go:69]: We could call `GetPxOperatorDeployment(..)` here.

File: test/integration_test/utils/px_operator.go [hunk 3]
```diff
@@ -60,3 +63,60 @@ func getPXOperatorImageTag() (string, error) {
...
+// UpdatePxOperator update PX Operator deploymnent
+func UpdatePxOperator(pxOperator *appsv1.Deployment) (*appsv1.Deployment, error) {
+	logrus.Infof("Update PX Operator %s in %s", pxOperator.Name, pxOperator.Namespace)
+
+	pxOperator, err := appops.Instance().UpdateDeployment(pxOperator)
+	if err != nil {
+		return nil, err
+	}
+
+	return pxOperator, nil
```
Comment (piyush-nimbalkar) [inline test/integration_test/utils/px_operator.go:121]: same here - you could just write `return appops.Instance().UpdateDeployment(pxOperator)`, if you want.

**Review by piyush-nimbalkar — APPROVED** [review]

File: test/integration_test/utils/px_operator.go (hunk 1 above)
Comment (nikolaypopov) [reply test/integration_test/utils/px_operator.go]: Done

File: test/integration_test/utils/px_operator.go (hunk 2 above)
Comment (nikolaypopov) [reply test/integration_test/utils/px_operator.go]: Done

File: test/integration_test/utils/px_operator.go (hunk 3 above)
Comment (nikolaypopov) [reply test/integration_test/utils/px_operator.go]: Done

**Pull request merged**